lite/provider.go

```diff
@@ -1,103 +1,32 @@
 package lite
 
-// Provider is used to get more validators by other means.
-//
-// Examples: MemProvider, files.Provider, client.Provider, CacheProvider....
+import (
+	"github.com/tendermint/tendermint/types"
+	log "github.com/tendermint/tendermint/libs/log"
+)
+
+// Provider provides information for the lite client to sync validators.
+// Examples: MemProvider, files.Provider, client.Provider, CacheProvider.
 type Provider interface {
```

Comment:
Why Certifier is defined in file types.go and Provider in provider.go?

Reply:
TODO make issue.  Leaving as is now to preserve history, until things have settled.